examples/TimeAwareSensor/TimeAwareSensor.ino

```diff
@@ -51,20 +51,22 @@ void presentation()  {
 }
 
 // This is called when a new time value was received
-void receiveTime(unsigned long time) {
+void receiveTime(uint32_t time) {
   // Ok, set incoming time 
   setTime(time);
   timeReceived = true;
+  Serial.println("Time received");
 }
  
 void loop()     
 {     
   unsigned long now = millis();
 
   // If no time has been received yet, request it every 10 second from controller
-  // When time has been received, request update every hour
+  // When time has been received, request update every 20 seconds 
+  // (use a more reasonable time like 60 minutes for a release)
   if ((!timeReceived && (now-lastRequest) > (10UL*1000UL))
-    || (timeReceived && (now-lastRequest) > (60UL*1000UL*60UL))) {
+    || (timeReceived && (now-lastRequest) > (20UL*1000UL*60UL))) {
```

Comment:
I missed this one: Why would you reduce the update interval to 20min?

Reply:
actually it was easier for me to test the sketch with 20 seconds time re-requesting, but forgot to remove the *60 UL for the minutes conversion.... 

As the sketch is meant for testing, I suggest to use (20UL *1000UL) (20 seconds delay) instead of the 60 minutes. What do you think?

Reply:
Since the update request is no different from the initial request, I'd leave the interval at 60min (the clock shift on the atmega328p is not that bad)